.gitignore

```
@@ -104,3 +104,6 @@ dist
 .tern-port
 
 /e2e-output/
+
+# Editor configs 
+.vscode
```

vscode user here so I need to ignore project settings

---

:+1: 

I think a per-user `.gitignore` is good for that kind of stuff, but good to add it to make contributing easier also.

Would add a trailing slash assuming it's a directory:

```suggestion
.vscode/
```

I think we need to add a `.editorconfig` file to ensure trailing newlines are added to files as well e.g.:

```
root = true

[*]
insert_final_newline = true
charset = utf-8
```